doc/tutorial/statistical_inference/settings.rst

```diff
@@ -51,6 +56,20 @@ needs to be preprocessed in order to be used by scikit-learn.
 
         >>> data = digits.images.reshape((digits.images.shape[0], -1))
 
+    .. raw :: html
```

Comment:
the result is kinda funny, I guess the image after the code block is enough?
![Screenshot from 2020-07-14 15-06-43](https://user-images.githubusercontent.com/1663864/87429213-b6012d80-c5e3-11ea-84d9-5405f841dcdb.png)

Reply:
Sorry, this is due to my bad merging with  #17919. Fixed.